scripts/changed_components.rb

```diff
@@ -87,6 +89,11 @@ def get_hab_scaffolding(source_dir)
   end
 end
 
+FORCE_SKIP_PACKAGES.each do |skip_pkg|
+  changed_packages.delete(skip_pkg)
+  changed_packages_reasons.delete(skip_pkg)
+end
```

Comment:
This change is excluding workflow from changed_components. This is good because [workflow is EOL December 31, 2020](https://blog.chef.io/new-chef-automate-functionality-and-end-of-life-impacts/) and there is some SCSS import issue that blocks it from building now. We _probably_ won't need to build it again. But if we do then this will probably confuse someone. I'm open to a better way to do it.